Gemfile.lock

```diff
@@ -322,10 +321,12 @@ DEPENDENCIES
   simplecov-rcov (~> 0.2.3)
   slimmer (= 8.2.1)
   smartdown (~> 0.15.1)
-  syck (= 1.0.5)
   therubyracer (~> 0.12.1)
   tilt (= 1.4.1)
   timecop
   uglifier
   unicorn (= 4.8.3)
   webmock (= 1.20.4)
+
+BUNDLED WITH
+   1.10.5
```

Comment:
I've heard of people having problems with different projects having different values of `BUNDLED WITH`, but I may be worrying about nothing.

When you say you "got the newest bundler version with it", Bundler isn't part of Ruby is it? i.e. presumably you got the latest version because you did `gem install bundler` in your freshly installed Ruby version. If so, this still feels like a separate change to me. You *could* prescribe the same version of Bundler as you were using in the earlier install of Ruby when you install the gem.

I prefer to keep changes like this as small as possible, because then if there's a problem it's easier to to work out where it's been introduced.